RegistryManager/src/main/java/eu/neclab/ngsildbroker/registryhandler/controller/RegistryController.java

```diff
@@ -83,7 +83,14 @@ public Uni<RestResponse<Object>> queryCSource(HttpServerRequest request, @QueryP
 		}
 		if (ids == null && type == null && attrs == null && geometry == null && q == null) {
 			return Uni.createFrom()
-					.item(HttpUtils.handleControllerExceptions(new ResponseException(ErrorType.InvalidRequest)));
+					.item(HttpUtils.handleControllerExceptions(new ResponseException(ErrorType.BadRequestData)));
```

Comment:
Validate this Exception as above comment all three point.

Reply:
done

Reply:
done